.dockerignore

```diff
@@ -0,0 +1,4 @@
+*
+!entrypoint.sh
+!requirements.txt
```

Comment:
Reduce the number of rebuilds; because almost all of the source is not in the Docker anymore, only depend on files that are actually needed.